components/Dashboard/candidate/profile/update-skills-modal.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState,KeyboardEvent } from 'react';
+import Modal from '@/components/modal/Modal';
+import { CancelIcon } from '@/assets';
+import { Button, OutlineBtn } from '@/components/button';
+interface modalcompProps {
+    open: boolean;
+    setOpen: any;
+    onClick: any;
+    loading: boolean;
+    setData: any;
+    data: any;
+}
+
+
+
+
+
+const UpdateSkillModal : React.FC<modalcompProps> = ({ open, setOpen, onClick, setData, loading,data }) => {
+    const [keyword, setKeyword] = useState("");
+    const [list, setList] = useState<any>([]);
+  
+    useEffect(() => {
+       
+
+
+        // setData(data);
+        setList(data)
+
+
+    }, [data])
+  
+
+    const handleKeyDown = (event: KeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            const newList:any = [...list];
+            newList.push(keyword);
+            setKeyword("");
+            setList(newList)
+            setData(newList);
+        }
+    };
+
+
+
+
+
+    return (
+        <div className='relative'>
+            <Modal open={open} setOpen={setOpen} >
+                <div className="w-full flex flex-col ">
+                    <div className="flex justify-end">
+                        <span onClick={() => setOpen(false)}>
+                            <CancelIcon />
+                        </span>
+                    </div>
+                    <div className="lg:w-10/12 w-full mx-auto mt-4 flex flex-col">
+                        <h1 className='text-black text-2xl lg:text-4xl font-bold'>
+                            Update Skills
+                        </h1>
+
+                        <div className="w-full mt-8    gap-4 flex flex-col  ">
+                            <label htmlFor="">
+                                Add skill set 
+                            </label>
+                            <div className="w-full  h-[56px] rounded-[15px] px-2 flex items-center gap-2 border border-[#63685E]">
```

Comment:
Remove arbitrary values

```suggestion
                            <div className="w-full  h-14 rounded-2xl px-2 flex items-center gap-2 border border-[#63685E]">
```